### ISSUE TITLE
[Request] Anti afk

### ISSUE THREAD

**Issue by TrashPandaCodingGarbage** [sergiorosendo/tibia_no_die#16]
Hello,

I am often skilling with mad sheep, and I am lacking a proper anti-afk solution.

This could be a simple script, to do a "dance" (ctrl+up, ctrl+right, ctrl+left etc/) at random intervals (between 10-13 minutes). 

Is it possible?

It will save my life! (In game, not literally!) :) Thanks!

**Comment by sergiorosendo** [comment]
Hi,

you can accomplish that using that auto-rune feature.

In config.ini, leave the keys for rune, food, ring and boots empty, and set mana_waste to a hotkey that triggers a spell not available to your vocation.

This way your character will attempt to cast a spell in intervals lower than 15 minutes, allowing it to stay online.

**Comment by TrashPandaCodingGarbage** [comment]
Thank you for your reply.
As far as I know, performing only 1 action in 15 minutes can get you kicked.

I have set up a macro, which only job was to mimic single mouse click, and click every 10 minutes, on a hotkey where I have assigned food. Still got kicked.

I measured a time, and indeed my macro was set up properly. It is possible that Battleye was kicking me, but I doubt it.

**Comment by TrashPandaCodingGarbage** [comment]
@sergiorosendo Indeed, I did what you said using NoDie script. Key was pressed, but I still got kicked.
